src/slash_command.ts

```diff
@@ -40,16 +43,14 @@ export async function handleSlashCommand(
         tfcCli.tfInit();
 
         // handle input vars here
-        let workspaceId = await tfcApi.getWorkspaceId();
-        console.log(`workspaceId = ${workspaceId}`);
-        let existingVars = await tfcApi.getExistingVars(workspaceId);
+        let existingVars = await tfcApi.getExistingVars();
```

Comment:
I removed workspaceId as an argument to setVariable since workspace IDs are unique to TFC. Instead TerraformCloudAPI#getExistingVars(...) manages the step of retrieving the TFC workspace ID.